includes/MediaWiki/Api/SingleEditEndpoint.php

```diff
@@ -59,29 +50,15 @@ public function run() {
 			throw new LocalizedHttpException( $rex->getMessageValue(), 400 );
 		}
 
-		$reconcileUrlProperty = $request->reconcilePropertyId();
-		$inputEntity = $request->entity();
-		$otherItems = $request->otherItems();
-
-		// Reconcile the item
-		try {
-			$reconciledItem = $this->itemReconciler->reconcileItem(
-				$inputEntity,
-				$reconcileUrlProperty
-			);
-		} catch ( ReconciliationException $rex ) {
-			throw new LocalizedHttpException( $rex->getMessageValue(), 400 );
-		}
-
-		$saveStatus = $this->persistItem( $reconciledItem, $otherItems );
+		$saveStatus = $this->editRequestSaver->persistEdits( [ $request ], $this->editToken, $this->user );
 
 		// Make some sort of response
 		$response = [
 			'success' => $saveStatus->isGood()
 		];
 		if ( $saveStatus->isGood() ) {
 			/** @var EntityRevision $entityRevision */
-			$entityRevision = $saveStatus->getValue()['revision'];
+			$entityRevision = $saveStatus->getValue()[0]['revision'];
```

Comment:
this might need a check for that element maybe?

Reply:
Or maybe not, if we reach this point and there is nothing here It should crash and burn probably.

Reply:
:+1: